ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2297,11 +2296,11 @@ qos_profiles:
     dscp: < dscp-value >
     tx_queues:
       < tx-queue-id >:
-        bandwidth_percent: < value >
-        priority: < string >
+        bandwidth_percent: < 1-100 >
+        priority: < "priority strict" | "no priority" >
       < tx-queue-id >:
```

Comment:
I think it's better to remove the lines 2301-2303. This is duplicating the same data.

Let's see what others will say. Same for above <tx-queue-id>.

We should try to minimize the duplication from now. Anyway, by specifying ```<tx-queue-id>``` the user might know the parent knob is supporting dictionaries.